pkg/eventpersisterdwh/storage/v2/sql/count_experiment.sql

```diff
@@ -0,0 +1,4 @@
+select count(*)
+from experiment
+where deleted = 0
```

Comment:
The conditions must match the conditions when listing the experiments.
The status and the 2-day buffer after the experiment has stopped.

Because there are conditions to list the experiments, please use this function.
https://github.com/bucketeer-io/bucketeer/blob/main/pkg/eventpersisterdwh/persister/evaluation_event.go#L227-L279

Reply:
Can you try this?
The current condition will get too much data unnecessarily.

```sql
SELECT (SELECT count(*) FROM experiment WHERE status = 1) + (SELECT count(*) FROM experiment WHERE status = 2 and DATE_ADD(from_unixtime(stop_at), INTERVAL 2 DAY) > NOW() ) as count;
```

Reply:
Looks good, let me try it!

Reply:
I think we can use this sub query to count stopped and still in 2-days buffer experiments:
```sql
SELECT count(*) 
FROM experiment 
WHERE status = 2 
  AND stop_at > UNIX_TIMESTAMP(DATE_SUB(NOW(), INTERVAL 2 DAY))

```
So we will not do calculation on `stop_at` column.

Reply:
Sounds good!